### PR TITLE
[SYCL][CUDA] Improve CUDA backend documentation

### PR DESCRIPTION
**Pull request by Ruyk**
-  Documented CMake option
-  Getting Started Guide clarifications

Co-Authored-By: Alexander Johnston <alexander@codeplay.com>
Signed-off-by: Ruyman Reyes <ruyman@codeplay.com>

### CODE REVIEW

File: sycl/doc/GetStartedGuide.md [hunk 1]
```diff
@@ -123,10 +123,15 @@ should be used.
...
-the CMake command when building the DPC++ compiler.
+To enable support for CUDA devices, follow the instructions for the Linux
+DPC++ toolchain, but replace the cmake command with the following one:
+
 
 ```
+cmake -DCMAKE_BUILD_TYPE=Release \
+-DLLVM_EXTERNAL_PROJECTS="llvm-spirv;sycl" \
+-DLLVM_EXTERNAL_SYCL_SOURCE_DIR=$SYCL_HOME/llvm/sycl \
+-DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR=$SYCL_HOME/llvm/llvm-spirv \
```
Comment (romanovvlad) [inline sycl/doc/GetStartedGuide.md]:
```suggestion
-DLLVM_EXTERNAL_SYCL_SOURCE_DIR=$DPCPP_HOME/llvm/sycl \
-DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR=$DPCPP_HOME/llvm/llvm-spirv \
```

File: sycl/doc/GetStartedGuide.md [hunk 2]
```diff
@@ -145,6 +150,24 @@ above.
 
 # Use DPC++ toolchain
 
+## Using the SYCL toolchain on CUDA platforms
```
Comment (romanovvlad) [inline sycl/doc/GetStartedGuide.md]:
```suggestion
## Using the DPCPP toolchain on CUDA platforms
```

File: sycl/doc/GetStartedGuide.md [hunk 3]
```diff
@@ -145,6 +150,24 @@ above.
...
+
+The SYCL toolchain support on CUDA platforms is still in an experimental phase.
```
Comment (romanovvlad) [inline sycl/doc/GetStartedGuide.md]:
```suggestion
The DPCPP toolchain support on CUDA platforms is still in an experimental phase.
```

Comment (romanovvlad) [reply sycl/doc/GetStartedGuide.md]: Please, replace other SYCL usages with DPCPP

Comment (pvchupin) [reply sycl/doc/GetStartedGuide.md]: DPC++ looks better!

File: sycl/doc/GetStartedGuide.md (hunk 2 above)
Comment (pvchupin) [reply sycl/doc/GetStartedGuide.md]: DPC++ please where it's human-only readable

File: sycl/doc/GetStartedGuide.md (hunk 1 above)
Comment (pvchupin) [reply sycl/doc/GetStartedGuide.md]: Thanks. Missed that.

File: sycl/doc/GetStartedGuide.md (hunk 3 above)
Comment (keryell) [reply sycl/doc/GetStartedGuide.md]: SYCL DPC++ looks better :-)

Comment (Ruyk) [reply sycl/doc/GetStartedGuide.md]: > SYCL DPC++ looks better :-)

Bit of a repetition there, right? ;-)

**Comment by Ruyk** [comment]
Thanks for the review, I've addressed everything (hopefully)

**Pull request merged**